js/src/forum/components/PostStreamScrubber.js

```diff
@@ -29,26 +31,26 @@ export default class PostStreamScrubber extends Component {
       count: <span className="Scrubber-count">{formatNumber(count)}</span>,
     });
 
-    const unreadCount = this.stream.discussion.unreadCount();
+    const unreadCount = this.attrs.discussion.unreadCount();
     const unreadPercent = count ? Math.min(count - this.stream.index, unreadCount) / count : 0;
 
-    function styleUnread(element, isInitialized, context) {
-      const $element = $(element);
+    function styleUnread(vnode) {
+      const $element = $(vnode.dom);
       const newStyle = {
         top: 100 - unreadPercent * 100 + '%',
         height: unreadPercent * 100 + '%',
       };
 
-      if (context.oldStyle) {
-        $element.stop(true).css(context.oldStyle).animate(newStyle);
+      if (vnode.oldStyle) {
+        $element.stop(true).css(vnode.oldStyle).animate(newStyle);
       } else {
         $element.css(newStyle);
       }
 
-      context.oldStyle = newStyle;
+      vnode.oldStyle = newStyle;
```

Comment:
Shouldn't this be stored on `vnode.state` and not `vnode` itself? According to https://mithril.js.org/vnodes.html#structure, `state` is persisted. The `vnode` is recreated on every render, I would assume.